fake_switches/brocade/command_processor/enabled.py

```diff
@@ -67,7 +67,7 @@ def do_skip_page_display(self, *args):
         pass
 
     def do_write(self, *args):
-        pass
+        self.switch_configuration.commit()
```

Comment:
Similar to what @mat128 said. The main idea here is that the commit is not only doing a delay. It can support other functionalities as well that in the future might be universal to all switches in its commit operation. 

If we feel like we should really limit this implementation to strictly a timeout. Then maybe we need to take the approach you suggested.

So the question for this would be: Which way should we go